gen-ai/orchestrator-server/src/main/python/server/src/gen_ai_orchestrator/models/rag/rag_models.py

```diff
@@ -15,40 +15,49 @@
 """Module for RAG Models"""
 
 from enum import Enum, unique
-from typing import Optional, List
+from typing import List, Optional
 
 from pydantic import AnyUrl, BaseModel, Field, HttpUrl
 
-from gen_ai_orchestrator.models.vector_stores.vector_stores_types import DocumentSearchParams
+from gen_ai_orchestrator.models.vector_stores.vector_stores_types import (
+    DocumentSearchParams,
+)
 
 
-class Footnote(BaseModel):
-    """A footnote model, used to associate document sources with the RAG answer"""
+class Source(BaseModel):
+    """A source model, used to associate document sources with the QA response"""
 
-    identifier: str = Field(description='Footnote identifier', examples=['1'])
-    title: str = Field(description='Footnote title', examples=['Tock Documentation'])
+    title: str = Field(description='Source title', examples=['Tock Documentation'])
     url: Optional[AnyUrl] = Field(
-        description='Footnote url', examples=['https://doc.tock.ai/tock/'], default=None
+        description='Source url', examples=['https://doc.tock.ai/tock/'], default=None
+    )
+    content: str = Field(
+        description='Source content', examples=['Tock: The Open Conversation Kit']
     )
-    content: str = Field(description='Footnote content', examples=['Tock: The Open Conversation Kit'])
 
     def __eq__(self, other):
         """
-        Footnotes are identified by their title and URL.
-        When the identifier or the content are not the same for identical footnotes,
+        Source are identified by their title and URL.
+        When the identifier or the content are not the same for identical sources,
         this means that the sources are parts of the same document.
         """
         return (
-                isinstance(other, Footnote)
-                and self.title == other.title
-                and self.url == other.url
-                and self.content == other.content
+            isinstance(other, Footnote)
```

Comment:
Copy / Past mistake : 
```suggestion
            isinstance(other, Source)
```